sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/storage/SQLiteUtils.java

```diff
@@ -8,41 +8,20 @@
 import android.content.ContentValues;
 import android.database.sqlite.SQLiteDatabase;
 import android.database.sqlite.SQLiteQueryBuilder;
+import android.database.sqlite.SQLiteStatement;
 import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
 
 import java.util.Map;
 
 public class SQLiteUtils {
-
     @NonNull
     public static SQLiteQueryBuilder newSQLiteQueryBuilder() {
         return new SQLiteQueryBuilder();
     }
 
-    public static void createTable(SQLiteDatabase db, String table, ContentValues schema) {
-
-        /* Generate a schema from specimen. */
-        StringBuilder sql = new StringBuilder("CREATE TABLE IF NOT EXISTS `");
-        sql.append(table);
-        sql.append("` (oid INTEGER PRIMARY KEY AUTOINCREMENT");
-        for (Map.Entry<String, Object> col : schema.valueSet()) {
-            sql.append(", `").append(col.getKey()).append("` ");
-            Object val = col.getValue();
-            if (val instanceof Double || val instanceof Float) {
-                sql.append("REAL");
-            } else if (val instanceof Number || val instanceof Boolean) {
-                sql.append("INTEGER");
-            } else if (val instanceof byte[]) {
-                sql.append("BLOB");
-            } else {
-                sql.append("TEXT");
-            }
-        }
-        sql.append(");");
-        db.execSQL(sql.toString());
-    }
-
-    public static void dropTable(@NonNull SQLiteDatabase db, @NonNull String table) {
-        db.execSQL(String.format("DROP TABLE `%s`", table));
+    public static void executeSQL(SQLiteDatabase db, String sqlCreateCommand) {
+        SQLiteStatement stmt = db.compileStatement(sqlCreateCommand);
+        stmt.execute();
```

Comment:
Why don't we use `execSQL`? It uses `SQLiteStatement` internally but also does some syncs if needed.